nautobot/core/views/mixins.py

```diff
@@ -349,6 +355,8 @@ def get_table_class(self):
             return NoteTable
         elif self.action == "changelog":
             return ObjectChangeTable
+        elif self.action == "data_compliance":
+            return DataComplianceTableTab
```

Comment:
Not clear to me if this is needed, but following notes/changelog pattern presuming that this is covering an edge case I had not considered.